`auto shipit --verbose` failing

**Describe the bug**

When attempting to upgrade Arty's auto installation to v9.10.4, I ran across a weird failure where `auto shipit --verbose` fails

Here's the error message

```
Error: Cannot find module '/home/circleci/project/--version'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:634:15)
    at Function.Module._resolveFilename (pkg/prelude/bootstrap.js:1287:46)
    at Function.Module._load (internal/modules/cjs/loader.js:560:25)
    at Function.Module.runMain (pkg/prelude/bootstrap.js:1316:12)
    at startup (internal/bootstrap/node.js:320:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:660:3)
{ Error: Command failed: /usr/local/bin/auto --version
internal/modules/cjs/loader.js:636
    throw err;
    ^

Error: Cannot find module '/home/circleci/project/--version'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:634:15)
    at Function.Module._resolveFilename (pkg/prelude/bootstrap.js:1287:46)
    at Function.Module._load (internal/modules/cjs/loader.js:560:25)
    at Function.Module.runMain (pkg/prelude/bootstrap.js:1316:12)
    at startup (internal/bootstrap/node.js:320:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:660:3)

    at checkExecSyncError (child_process.js:629:11)
    at Object.execSync (child_process.js:666:13)
    at Object.childProcess.execSync (pkg/prelude/bootstrap.js:1448:30)
    at Auto.info (/snapshot/auto/node_modules/@auto-it/core/dist/auto.js:260:41)
```

**To Reproduce**

I haven't verified yet that it always happens when the verbose flag is used, but I SSH'd into the failing job on circle and ran without it successfully. 

Here's a link to the failing build: https://app.circleci.com/jobs/github/artsy/reaction/5172

--- a/packages/core/src/auto.ts
+++ b/packages/core/src/auto.ts
@@ -521,8 +521,10 @@ export default class Auto {
       ${chalk.underline.white('Environment Information:')}
 
       "auto" version: v${getAutoVersion()}
-      "node" version: ${execSync('node --version', { encoding: 'utf8' }).trim()}
-      ${access['x-github-enterprise-version'] ? `GHE version:    v${access['x-github-enterprise-version']}\n`: '\n'}
+      "node" version: ${process.version.trim()}${
+        access['x-github-enterprise-version'] 
+          ? `GHE version:    v${access['x-github-enterprise-version']}\n`
+          : '\n'}
       ${chalk.underline.white('Project Information:')}
 
       ${logSuccess(noProject)} Repository:      ${repoLink}

